pkgs/top-level/python-packages.nix

```diff
@@ -11890,6 +11890,20 @@ in modules // {
     };
   });
 
+  libgpuarray-cuda = callPackage ../development/python-modules/libgpuarray/cuda/default.nix rec {
+    numpy = self.numpy_1_10;
+    scipy = self.buildScipyPackage rec {
+      version = "0.17.1";
+      src = pkgs.fetchurl {
```

Comment:
The final `scipy` here is not exactly the same as if you take `self.scipy`. The problem is that one needs `scipy` built with `numpy_1_10`, not with `numpy_1_11`. The definition of `scipy_0_17` contains `numpy = self.numpy` in place of `inherit numpy`.

In principle, `libgpuarray` can be built without `scipy` as a dependency. `scipy` is only needed to run the tests afterwards. The reason I take `numpy_1_10` is to pass *all* the tests. If you take `numpy_1_11`, then everything looks fine except for the three tests that fail:
```
pygpu.tests.test_operations.test_xsplit ('h', (4, 4), [3, 6])
pygpu.tests.test_operations.test_xsplit ('v', (4, 4), [3, 6])
pygpu.tests.test_operations.test_xsplit ('d', (4, 4), [3, 6])
``` 
This is probably the same issue as https://github.com/Theano/libgpuarray/issues/184 .

It might be the case that `libgpuarray` built with `self.sumpy` and `self.scipy` works fine with `Theano`, i.e. that all `Theano` tests succeed. I am going to check it.